catboost/docs/en/concepts/parameter-tuning.md

```diff
@@ -479,3 +479,12 @@ Class | Method | Description
 [CatBoostRegressor](python-reference_catboostregressor.md) | [randomized_search](python-reference_catboostregressor_randomized_search.md) | A simple randomized search on hyperparameters. |
 
 {% endcut %}
+
+## Methods for hyperparameter search by optuna
+
+Optuna is a famous hyperparameter optimization framework.
+Optuna enables efficient hyperparameter optimization by adopting state-of-the-art algorithms for sampling hyperparameters and pruning efficiently unpromising trials.  
+Catboost supports to stop unpromising trial of hyperparameter by callbacking after iteration functionality. (https://github.com/catboost/catboost/pull/1697)
```

Comment:
Links are not rendered automatically and it is better to put the direct link to the example from this pull request.
`[Example](https://github.com/catboost/catboost/pull/1697/files#diff-ccca44461ac6b094190f29fec157a227996e226ea483213680dd0a152cd412eaR9679)`